src/main/java/picard/illumina/parser/readers/CbclReader.java

```diff
@@ -333,38 +336,11 @@ private void cacheTile(final int totalCycleCount, final TileData tileData, final
         byte[] decompressedByteArray = decompressTile(totalCycleCount, tileData, byteInputStream);
 
         // Read uncompressed data from the buffer and expand each nibble into a full byte for ease of use
-        byte[] unNibbledByteArray = promoteNibblesToBytes(decompressedByteArray);
-        cachedTile[totalCycleCount] = filterNonPfReads(tileData, currentCycleData, unNibbledByteArray);
+        cachedTile[totalCycleCount] = promoteNibblesToBytes(decompressedByteArray);
 
         cachedTilePosition[totalCycleCount] = 0;
     }
 
-    private byte[] filterNonPfReads(TileData tileData, CycleData currentCycleData, byte[] unNibbledByteArray) {
-        // Write buffer contents to cached tile array
-        // if nonPF reads are included we need to strip them out
-        if (!currentCycleData.pfExcluded) {
```

Comment:
This appears to be the only use of the field `CycleData.pfExcluded`. if so, the field should be removed.

Reply:
This is parsed from the header.. I think it may be useful to keep it here in case it is ever needed and to document the header format. Otherwise we are parsing it and throwing it away. (which i suppose is fine too)